javascript/house-style.md

```diff
@@ -699,23 +699,22 @@ When it comes time to attach your [JavaScript module](#module-architecture) to a
 We do this:
 
 ```html
-<nav class="c-popout-nav c-popout-nav--dark" data-component="popout">
-  <button class="c-popout-nav__close" data-component="popout-close">Close</button>
-  (etc.)
+<button class="c-button" data-component-collapse-toggle="header-menu">Menu</button>
+<nav class="c-header-menu u-hidden" data-component-collapse-togglable="header-menu">
```

Comment:
Can we think about these examples? Because for me `data-component-collapse-toggle` and `data-component-collapse-togglable` don't say what they do.

Maybe something more like `data-component-collapse-activator` and `data-component-collapse-target`?